survey-runner-database/global_vars.tf

```diff
@@ -21,6 +21,7 @@ variable "vpc_id" {
 }
 
 variable "private_route_table_ids" {
+  type = "list"
   description = "Route tables with route to NAT gateway"
```

Comment:
When I run the `run-terraform.sh` script I get an error:
`variable private_route_table_ids should be type list, got string`